examples/creating_a_gwcs_from_quantities.py

```diff
@@ -0,0 +1,55 @@
+"""
+===============================================
+How to create an gWCS from quantities and times
```

Comment:
```suggestion
How to create an GWCS from quantities and times
```
I was wrong, the gwcs docs, call it GWCS.